src/reconcile/rec_visibility.c

```diff
@@ -150,6 +149,25 @@ __rec_append_orig_value(
             WT_ASSERT(session, F_ISSET(unpack, WT_CELL_UNPACK_PREPARE) ||
                 (unpack->tw.stop_ts == oldest_upd->start_ts &&
                                  unpack->tw.stop_txn == oldest_upd->txnid));
+            if (tombstone_globally_visible)
```

Comment:
Why don't we care about the unpacked value's stop timestamp here (like we do in the case where the oldest update isn't a tombstone)?

Reply:
I don't get your question here. tombstone_globally_visible checks the stop time pair of the unpack.

Reply:
I get your question now. Because the tombstone is already on the update chain so no need to worry about that.